src/runtime/transition/index.ts

```diff
@@ -3,6 +3,8 @@ import { assign, is_function } from 'svelte/internal';
 
 type EasingFunction = (t: number) => number;
 
+type SlideDirections = "vertical" | "horizontal"
```

Comment:
can you rebase against master and make sure `npm run lint` passes? we just updated our eslint config to allow only single quotes

Reply:
done